packages/checkbox/src/mwc-checkbox-base.ts

```diff
@@ -167,32 +169,43 @@ export class CheckboxBase extends FormElement {
       </div>`;
   }
 
-  private _handleFocus() {
-    this.focused = true;
-    this.rippleHandlers.startFocus();
-  }
+  @eventOptions({passive: true})
```

Comment:
This line can probably be removed, as I don't think making `mousedown` passive affects anything.